src/guide/1/getting-started/node-js/index.md

```diff
@@ -0,0 +1,251 @@
+---
+layout: full.html.hbs
+algolia: true
+title: NodeJS
+description: Getting started with Kuzzle and Node.js
+---
+
+# Getting Started with Kuzzle and Node.js
+
+In this tutorial you will learn how to install, run and use **Kuzzle** with **Node.js** and the **Javascript SDK**.
+We will walk you through creating scripts that can **store** documents in Kuzzle and subscribe to **notification** for each new document created.
+
+## Running Kuzzle
+
+In this section we'll learn how to quickly get Kuzzle up and running using our installation script.
+
+Open a terminal and run the following command:
+
+```bash
+bash -c "$(curl https://get.kuzzle.io/)"
+```
+
+This command downloads and executes the installation script. The script checks the system for a set of prerequisites and installs any necessary tools, like Docker or Docker Compose. When the installation is complete it will automatically run Kuzzle.
```

Comment:
```suggestion
This command downloads and executes the installation script. The script checks the system for a set of prerequisites, like Docker or Docker Compose. If something's missing, the script will give you some hints on how to install them. When the installation is complete it will automatically run Kuzzle.
```